backend/src/main/java/com/crm/model/db/CustomerEntity.java

```diff
@@ -34,16 +34,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @NotNull
     private String name;
```

Comment:
Why @NotNull annotations were deleted?

Reply:
Because the fields are validated in CustomerRequest class

Reply:
Ok